Update cloud doc for trends graphs fix (#5545)
mendix/docs: content/en/docs/releasenotes/deployment/mendix-cloud.md

## 2023

### January 18th, 2023

> #### Improvements

Reviewer: Is this an improvement, or a fix?
Have changed to improvement to match the text.

Author: It's actually a fix for regression that we introduced after the rollout of Mendix 9.7. So I'd prefer fix

Reviewer: ```suggestion
#### Fix
```

* We have fixed scaling issues on graphs for metrics **Number of handled external requests** and **Number of database queries being executed** 

> 

Reviewer 2: Should we talk something about the runtime version too?

something on these lines?

We have fixed the metrics graph **Number of handled external requests** and **Number of database queries being executed**  that was broken in the runtime version >= 9.7.0

Author: updating the sentence and adding a bit more explanation 👍

Reviewer 3: For customers running Mendix apps with runtime 9.7 and above, we have made improvements to **Number of handled external requests** and **Number of database queries being executed** graphs. These changes are available after you redeploy your app.

> * For customers running Mendix apps with runtime version 9.7.0 and above, we have made improvements to [Number of handled external requests](/developerportal/operate/trends-v4/#Trends-appmxruntimerequests) and [Number of database queries being executed](/developerportal/operate/trends-v4/#Trends-dbmxruntimeconnectionbus) graphs. These changes are available after you redeploy your app.

Reviewer: Do we need to update the trends documentation, or are the changes just to the data quality rather than the data which is displayed?

Author: It's a regression fix so don't need to update trends docs for this fix.

Reviewer: Hi @ekremsekerci 
If it isn't an improvement then I think we should match the terminology and perhaps say:
```suggestion
* For customers running Mendix apps with runtime version 9.7.0 and above, we have fixed some minor issues with the [Number of handled external requests](/developerportal/operate/trends-v4/#Trends-appmxruntimerequests) and [Number of database queries being executed](/developerportal/operate/trends-v4/#Trends-dbmxruntimeconnectionbus) graphs. These changes are available after you redeploy your app.
```
Does that look OK?

Author: Yeah, this is better. Thanks. We can merge this whenever is possible :)
